sam-styles/card/_card.scss

```diff
@@ -60,9 +60,10 @@
   color: #2e2e2a;
 }
 
-.basic-card--data {
+.basic-card--data, h5 {
```

Comment:
This targets all `h5` tags - please make more specific

Reply:
thanks! I made it more specific